sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -383,4 +421,21 @@ static inline bool _node_unreachable(_nib_onl_entry_t *node)
     }
 }
 
+uint32_t _evtimer_lookup(const void *ctx, uint16_t type)
+{
+    evtimer_msg_event_t *event = (evtimer_msg_event_t *)_nib_evtimer.events;
+    uint32_t offset = 0;
```

Comment:
1. If offset wraps around, the implementation of `evtimer` itself would be broken, because `evtimer_add()` expects offset as `uint32_t` as well.
2. NDP doesn't work on the timescale of weeks ((2³² - 1) ms > 7 weeks)